### PR TITLE
Has 72 adding functionalities for chronic illness entity

### PR DESCRIPTION
**Pull request by Dhruv-1105**
added functionalities for chronic illness entity

### CODE REVIEW

**Review by mjoshi-92 — APPROVED** [review]
Good

**Review by SanketRathod666 — APPROVED** [review]

File: Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/ChronicIllnessDaoImpl.java [hunk 1]
```diff
@@ -1,8 +1,11 @@
 package com.spring.hasdocTime.dao;
 
+import com.electronwill.nightconfig.core.conversion.SpecClassInArray;
```
Comment (a-bhavsar) [inline Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/ChronicIllnessDaoImpl.java]: Remove this

File: Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/UserDaoImpl.java [hunk 2]
```diff
@@ -36,7 +49,33 @@ public User getUser(int id) {
 
     @Override
     public User createUser(User user) {
-        user.setId(0);
+//        user.setId(0);
```
Comment (a-bhavsar) [inline Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/UserDaoImpl.java]: Remove this

File: Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/UserDaoImpl.java [hunk 3]
```diff
@@ -46,8 +85,34 @@ public User updateUser(int id, User user) {
         if(oldUser.isPresent()) {
             User oldUserObj = oldUser.get();
             user.setId(oldUserObj.getId());
-            userRepository.save(user);
-            return user;
+//            List<Symptom> symptoms = new ArrayList<>();
```
Comment (a-bhavsar) [inline Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/UserDaoImpl.java]: Remove this comment if not referring

**Review by a-bhavsar — APPROVED** [review]

**Review by a-bhavsar — APPROVED** [review]

**Pull request merged**